lib/aikido/zen/request.rb

```diff
@@ -40,6 +41,17 @@ def schema
       @schema ||= Aikido::Zen::Request::Schema.build
     end
 
+    # @api private
+    #
+    # @return [String] the IP address of the client making the request.
+    def client_ip
+      return @client_ip if @client_ip
+
+      @client_ip = env[@config.client_ip_header] if @config.client_ip_header
```

Comment:
```suggestion
      @client_ip = env[@config.client_ip_header] if @config.client_ip_header and env[@config.client_ip_header] =~ Resolv::AddressRegex
```

Validate if this is an IP address?

Reply:
We decided not to fail silently here.